mocks.go

```diff
@@ -116,6 +118,9 @@ func (r *Transport) RoundTrip(req *http.Request) (mockResponse *http.Response, m
 		fmt.Printf("failed to match mocks. Errors: %s\n", matchErrors)
 	}
 
+	if r.apiTest.mockMisses != nil {
```

Comment:
not sure if I'm missing something, but I think this will never be true. As the only place in your PR that you set a value to mockMisses is here, and you only set it if it is nil, and it defaults to nil, so can't see where it could be not nil